decentraland/screens/MyAssetsScreen.js

```diff
@@ -80,15 +80,15 @@ export class MyAssetsScreen extends React.Component {
       id => !fromBlockchainIds.includes(id)
     );
 
-    logWarn(
+    logInfo(
       `Account's assets from blockchain aren't the same as the ones stored on the app.`
     );
 
     if (addedIds.length > 0)
-      logWarn(`Some assets added to MyLandScreen. IDs: [${addedIds}]`);
+      logInfo(`Some assets added to MyLandScreen. IDs: [${addedIds}]`);
 
     if (removedIds.length > 0)
-      logWarn(`Some assets removed from MyLandScreen. IDs: [${removedIds}]`);
+      logInfo(`Some assets removed from MyLandScreen. IDs: [${removedIds}]`);
```

Comment:
Wouldn't we still want this one to be a warn, though?

Reply:
At first I thought this might as well come in this PR, but given that it's the end of the day, I'll merge the PR as-is and we can circle back to this.

Reply:
> Wouldn't we still want this one to be a warn, though?

Yes, make sense.

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/342

Reply:
Thanks! Resolving this